### ISSUE TITLE
mike doesnt support !ENV

### ISSUE THREAD

**Issue by zhilyaev** [jimporter/mike#64]
I have the next `mkdocs.yml`

```
...
extra:
  version:
    provider: mike
    default: latest
  analytics:
    provider: google
    property: !ENV GOOGLE_ANALYTICS_KEY
...
```

## Error

```
error: could not determine a constructor for the tag '!ENV'
  in "mkdocs.yml", line 53, column 15
```

Mkdocs configuration: https://www.mkdocs.org/user-guide/configuration/#environment-variables

**Comment by jimporter** [comment]
This should already be resolved by the patch for #59, which will be published in v1.1 in a week or so. If the patch there doesn't work right, just let me know and I can reopen this.

**Comment by jimporter** [comment]
Well, it took a lot longer than a week (other things came up, as they tend to do), but v1.1.0 is now published.
